client/src/utils.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 import React from 'react'
 import Loading from './components/Loading/Loading'
+import {AutoSizer} from 'react-virtualized'
```

Comment:
the lib is huge - https://bundlephobia.com/result?p=react-virtualized@9.20.1

is it worth pulling it in just for this functionality ? can't the same be done by using correct markup & css ?

Reply:
I can further investigate if the same can be easily achieved just with css/markup and will let you know.